code/modules/admin/verbs/randomverbs.dm

```diff
@@ -45,6 +45,33 @@
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Subtle Message") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_centcom_message(mob/M in GLOB.mob_list)
+	set category = "Special Verbs"
+	set name = "CentCom Message"
+	var/mob/living/carbon/human/H = M
+	
+	if(!check_rights(R_ADMIN))
+		return
+
+	if(!istype(H))
+		to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human")
+		return
+	if(!istype(H.ears, /obj/item/radio/headset))
+		to_chat(usr, "The person you are trying to contact is not wearing a headset.")
+		return
+
+	message_admins("[key_name_admin(src)] has started answering [key_name(H)]'s CentCom request.")
+	var/input = input("Please enter a message to reply to [key_name(H)] via their headset.","Outgoing message from CentCom", "") as text|null
+	if(!input)
+		message_admins("[key_name_admin(src)] decided not to answer [key_name(H)]'s CentCom request.")
+		return
+
+	log_admin("[key_name_admin(src)] replied to [key_name(H)]'s CentCom message with the message [input].")

```

Comment:
key_name_admin -> key_name

Reply:
Why is this one different from the other two?

Reply:
key_name_admin contains HTML, which doesn't belong in the logs. message_admins should always use it though